From 21e0bc0e62b25fd904d0d15fece5c44555cf872c Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Thu, 28 Nov 2024 15:41:42 +0800
Subject: [PATCH 2/3] quirks: add pressure pad quirk for Lenovo ThinkBook 14
 G6+ IMH

The ThinkBook 14 G6+ IMH also has the same issue as the ThinkBook 14 G7+ ASP.

Part-of: <https://gitlab.freedesktop.org/libinput/libinput/-/merge_requests/1083>
---
 quirks/50-system-lenovo.quirks | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/quirks/50-system-lenovo.quirks b/quirks/50-system-lenovo.quirks
index 6abbd368..30a79fc7 100644
--- a/quirks/50-system-lenovo.quirks
+++ b/quirks/50-system-lenovo.quirks
@@ -384,3 +384,10 @@ MatchName=*GXTP5100*
 MatchDMIModalias=dmi:*svnLENOVO:*pvrThinkBook14G7+ASP*:*
 MatchUdevType=touchpad
 ModelPressurePad=1
+
+# The ThinkBook 14 G6+ IMH also has a similar issue as the G7+ mentioned above.
+[Lenovo ThinkBook 14 G6+ IMH]
+MatchName=*GXTP5100*
+MatchDMIModalias=dmi:*svnLENOVO:*pvrThinkBook14G6+IMH*:*
+MatchUdevType=touchpad
+ModelPressurePad=1
-- 
2.47.1

